src/main/java/org/openrewrite/java/migrate/lang/RemoveDeprecatedRuntime.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.openrewrite.java.migrate.lang;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Preconditions;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.search.UsesType;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.TypeUtils;
+
+import java.util.Objects;
+
+public class RemoveDeprecatedRuntime extends Recipe {
+
+    public static final String TRACE_INSTRUCTIONS = "traceInstructions";
+    private static final String JAVA_LANG_RUNTIME = "java.lang.Runtime";
+    public static final String TRACE_METHOD_CALLS = "traceMethodCalls";
+
+    @Override
+    public String getDisplayName() {
+        return "Remove deprecated statements from Runtime module";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Remove deprecated invocations of Runtime.traceInstructions() and Runtime.traceMethodCalls() which have no alternatives needed.";
+    }
+
+    @Override
+    public TreeVisitor<?, ExecutionContext> getVisitor() {
+
+
+        return Preconditions.check(new UsesType<>(JAVA_LANG_RUNTIME, false),
```

Comment:
I had hoped to instead here use:
```java
        return Preconditions.check(Preconditions.or(new UsesMethod<>(TRACE_INSTRUCTIONS), new UsesMethod<>(TRACE_METHOD_CALLS)),
```
But that trips up the tests since those use Java 17 by default, which lacks the corresponding methods, and forcing `Java8Parser` or `Java11Parser` there so far gives the same result, as well as trying to override the Runtime class in a `java(...)` SourceSpec block. Any suggestions @knutwannheden ?

Reply:
With the following fields at the top:
```java
    private static final MethodMatcher TRACE_INSTRUCTIONS = new MethodMatcher("java.lang.Runtime traceInstructions(boolean)");
    private static final MethodMatcher TRACE_METHOD_CALLS = new MethodMatcher("java.lang.Runtime traceMethodCalls(boolean)");
```

Reply:
That indeed sounds tricky... It may be possible to set the `-release` compiler option in `ReloadableJava17Parser` and we may even want to add some of these options to the `JavaParser.Builder` API. I am not suggesting we do this in the scope of this PR, but this may allow us to test this properly in the future.

For now we may have to live with that we cannot properly test this.